packages/bootstrap-4/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -65,8 +65,8 @@ export default function FieldTemplate<
           </Form.Label>
         )}
         {children}
-        {displayLabel && rawDescription && (
-          <Form.Text className={rawErrors.length > 0 ? 'text-danger' : 'text-muted'}>{rawDescription}</Form.Text>
+        {displayLabel && description && (
+          <Form.Text className={rawErrors.length > 0 ? 'text-danger' : 'text-muted'}>{description}</Form.Text>
```

Comment:
`rawDescription` should continue to be used as the condition since it will be null when there is no description provided. This should cause most of the need to update the snapshots to go away when the description is an empty React element.
```suggestion
        {displayLabel && rawDescription && (
          <Form.Text className={rawErrors.length > 0 ? 'text-danger' : 'text-muted'}>{description}</Form.Text>
```

Reply:
Understood.